.github/workflows/test.yml

```diff
@@ -2,19 +2,17 @@ name: CI
 
 on:
   push:
-    branches:
```

Comment:
Why did you remove the testing on push to main?

Reply:
What this change does is run the workflow on all branch pushes, not just main. Without it, when I push my development branch to my repo, the workflow does not run. You can see a demo of that with the branch push test-workflow-branches-option at https://github.com/rkent/rosdoc2, which reverted this change. The workflow did not run, while the previous branch push with this change did.